### ISSUE TITLE
Error parsing Internet and Intranet Predefined set of computers

### ISSUE THREAD

**Issue by filippobottega** [Z3Prover/FirewallChecker#8]
Hello,
  in file WindowsFirewallRuleParser.cs the method `private static AddressSet ParseAddressSet(string text)` raises an exception in case of Internet and Intranet Predefined set of computers.
  Please see [Windows Firewall. Internet and Intranet Predefined set of computers.](https://social.technet.microsoft.com/Forums/windowsserver/en-US/4d2e4cce-ebc2-433c-8c7c-3e1bc5376471/windows-firewall-internet-and-intranet-predefined-set-of-computers?forum=winserversecurity).
  I suppose to be a bug.

  Regards,
    Filippo.

**Comment by ahelwer** [comment]
This is similar to port macros, which are currently [not supported](https://github.com/Z3Prover/FirewallChecker/blob/5dba05e2afd40f2422a733be959ff987e854c8f3/FirewallAnalysis/WindowsFirewallRuleParser.cs#L261); you can fix this by replacing the macros in your firewall input file with a comma-delimited set of addresses corresponding to the Internet macro. The code should probably handle this more gracefully, though (I didn't know there were macros for addresses as well as ports).

**Comment by filippobottega** [comment]
Yes, I agree with you, there are macros for addresses and I think that FirewallChecker has to take care about macro conversion to the current values. The problem is that there are a lot of rules that use macros and without macro conversion FirewallChecker may be not so useful.

**Comment by ahelwer** [comment]
What are you using firewall checker for?

**Comment by filippobottega** [comment]
I need a tool to understand which rules are allowing inbound traffic. For example I have had the problem to understand why my PC was able to response to ping requests.
